### ISSUE TITLE
Make the "edit" note a second optional step

### ISSUE THREAD

**Issue by seal20** [Fmstrat/ownnote#42]
When I open a note I expect it to be open "read only" with no editor frame. Then if i want to edit it, I will click an "edit button" toggling the edit mode.

Read only will provide a more distraction free reading experience, also links would be directly clickable without having to hold ctrl.

**Comment by marco9876** [comment]
Yeah thats nice... it also would solve the Problem with Checking Checkboxes... there is a Checkbox plugin but it doesnt work because in Tinymce its blocked!

**Comment by Fmstrat** [comment]
Funny how I'm getting comments both ways from users. Some want to edit right away, some don't. I'll add this as an enhancement in the admin section for allowing administrators to choose how they would like their server to function.

**Comment by seal20** [comment]
lol ! Yes an option will please everyone, but too much is not good too and it may make more work for you.

**Comment by Fmstrat** [comment]
True, but I've had a few users bring it up. We open source developers serve the people after all. :)

**Comment by mnlipp** [comment]
Is it at all possible to toggle between edit mode and read only mode? I haven't found a way to see my onwnnote in the browser in a read only mode. The big problem about this is following links. seal20 seems to have succeeded in following links by holding the ctrl key when clicking. somehow this doesn't work for me. I have to copy-paste the link in the note when I want to follow it, which makes onwnnote very tedious to use.

**Comment by milankni** [comment]
Yep, there is no way to follow a link in Chromium browser 45.0, whatever modified key I try to use (CTRL, SHIFT).
